DESCRIPTION

```diff
@@ -34,7 +34,7 @@ Suggests:
 VignetteBuilder: knitr
 RoxygenNote: 6.0.1
 Roxygen: list(markdown = TRUE)
-Collate:
+Collate: 
```

Comment:
Why?

Reply:
I had the same thing happen to me on a separate, I think it's a recent(ish) update to the way desc (called via roxygen2) deals with formatting `DESCRIPTION`s:

> Keep trailing space after the : character, see #14 
from 1.0.1 at https://github.com/r-lib/desc/blob/master/inst/NEWS.md

Roxygen2 using `desc` was supposed to help stop spurios `DESCRIPTION` changes:
> Changes to DESCRIPTION (i.e. Collate: and RoxygenNote) now use the desc package. This will minimise spurious changes (#430).
from 6.0.0 at https://github.com/klutometis/roxygen/blob/master/NEWS.md

If only because we will be fighting ourselves, I say let's adopt that convention / let `desc` / `roxygen` force it on us.